ssh-key/tests/public_key.rs

```diff
@@ -3,6 +3,11 @@
 use hex_literal::hex;
 use ssh_key::{Algorithm, PublicKey};
 use std::collections::HashSet;
+#[cfg(feature = "ecdsa")]
+use {
+    sec1::consts::U32,
+    ssh_key::public::{Ed25519PublicKey, SkEcdsaSha2NistP256, SkEd25519},
```

Comment:
Aah, you'll need to split out the import of `Ed25519PublicKey` so it's always imported and not gated on the `ecdsa` feature

Reply:
I see. Does the Ed25519PublicKey import need any gating as well?